Braintree.xcodeproj/project.pbxproj

```diff
@@ -684,6 +671,17 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		42F174F82576F4560092A66A /* Embed Frameworks */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				42F174F72576F4560092A66A /* PPRiskMagnes.framework in Embed Frameworks */,
+			);
+			name = "Embed Frameworks";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
```

Comment:
It looks like embedding the PPRiskMagnes framework was included in this PR. I assume it was left over from a previous uncommitted change?